flags/flags.go

```diff
@@ -110,13 +117,32 @@ func parse(args []string, terminalInfo terminalInfo) ([]string, Usage, *OptionSe
 	if err != nil {
 		return nil, nil, nil, err
 	}
+	if outputOptions.Download {
+		d = time.Duration(0)
+		exchangeOptions.FollowRedirects = true
+	}
 	exchangeOptions.Timeout = d
 
 	// Parse --pretty
 	if err := parsePretty(prettyFlag, terminalInfo.stdoutIsTerminal, &outputOptions); err != nil {
 		return nil, nil, nil, err
 	}
 
+	// Verify SSL
+	verifyFlag = strings.ToLower(verifyFlag)
+	switch verifyFlag {
+	case "no":
+		exchangeOptions.SkipVerify = true
+	case "yes":
+	case "":
+		exchangeOptions.SkipVerify = false
+	default:
+		return nil, nil, nil, fmt.Errorf("%s", "Verify flag must be 'yes' or 'no'")
+	}
+
+	// Force HTTP/1.1
+	exchangeOptions.ForceHTTP1 = http1Flag
```

Comment:
ForceHTTP1 flag is no longer used. Please remove this flag.

Reply:
You're right but I think being able to test h2 and http1 could be really useful (it was in my case). The original httpie can't do that natively but curl yes by adding "h2" flag. I'm looking for a solution. :)

Reply:
Working !

❯ go run ./cmd/ht -v -F --http1 https://httpbin.org
GET / HTTP/1.1
Accept-Encoding: gzip
Host: 
User-Agent: httpie-go/0.6.0


HTTP/1.1 200 OK
Access-Control-Allow-Credentials: true
Access-Control-Allow-Origin: *
Connection: keep-alive
Content-Length: 9593
Content-Type: text/html; charset=utf-8
Date: Fri, 20 Mar 2020 09:11:07 GMT
Server: gunicorn/19.9.0

❯ go run ./cmd/ht -v -F https://httpbin.org 
GET / HTTP/1.1
Accept-Encoding: gzip
Host: 
User-Agent: httpie-go/0.6.0


HTTP/2.0 200 OK
Access-Control-Allow-Credentials: true
Access-Control-Allow-Origin: *
Content-Length: 9593
Content-Type: text/html; charset=utf-8
Date: Fri, 20 Mar 2020 09:11:22 GMT
Server: gunicorn/19.9.0